verification/curator-service/src/controllers/home.ts

```diff
@@ -1,10 +1,37 @@
 import { Request, Response } from 'express';
+import axios from 'axios';
+
+interface Case {
```

Comment:
yeah that duplication is certainly not great once those types will start to grow and we'll have to keep them in sync :( cases are already defined in model/case.ts so let's find a way to export what we need from there. (in a later PR, this is fine for now)

Reply:
+1 -- we should discuss more